### DISCUSSION TITLE
How to List all Failed/Success package-name within task execution?

### DISCUSSION THREAD

**Discussion by ImBIOS** [vercel/turborepo#4758] [Help] [answered]
## Background Story

I'm using Turborepo to check whether all of the examples within my project are successfully built or not. I found it helpful to use "--continue --summarize" because even though there are several errors on several packages, I'm able to know how many packages failed.

but the summary in JSON format seems not very helpful, Is there any method I can use to get the list of successful or failed package names?

**Comment by anthonyshew** [comment]
The summary shown at the end of a `turbo` invocation lists out tasks that have failed. Is this a request to be able to programmatically access that data?

**Reply by ImBIOS** [reply, marked as answer]
I believe it's for one of my npm package project, but currently I forgot which one I have been experiencing with this problem. For now, let me close this discussion, since I'm always come up with workaround, I think around that time I have been working with workaround and happy since then.
